docs/api/url.md

```diff
@@ -0,0 +1,90 @@
+# 🔗 URL
+An API for working with [HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP) [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL)s. 
```

Comment:
This should be phrased as `jstz` implementing the URL spec.
Additionally URLs aren't defined by the HTTP specification.